inc/functions_upload.php

```diff
@@ -8,6 +8,29 @@
  *
  */
 
+/**
+ * Get maximum upload filesize limit set in PHP
+ * @since MyBB 1.8.27
+ * @return int maximum allowed filesize
+ */
+function get_phpuploadlimit()
+{	
+	$maxsize = array(return_bytes(ini_get('upload_max_filesize')), return_bytes(ini_get('post_max_size')));
+	$maxsize = array_filter($maxsize); // Remove empty values
+
+	if(empty($maxsize))
+	{
+		return 0;
+	}
+	else if(count($maxsize) == 1)
+	{
+		return (int)$maxsize[0];
```

Comment:
The `0` key may not be set as `array_filter()` can leave gaps in keys - we can use `min()` even if there is only one result

Reply:
I forgot that `min()` accepts single value through array.